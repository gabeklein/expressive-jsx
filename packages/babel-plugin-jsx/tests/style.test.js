transform("css module styles", `
  () => {
    color: red;

    <this />
  }
`, {
  externals: true,
  extractCss(css){
    return "./somefile.module.css"
  },
})

transform("absolute macro", `
  () => {
    absolute: fill;

    <this />
  }
`)