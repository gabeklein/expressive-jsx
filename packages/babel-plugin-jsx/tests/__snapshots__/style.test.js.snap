// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`absolute macro 1`] = `
  () => <div className="element_7nq" />;
  css(\`
  	.element_7nq {
  		position: absolute;
  		top: 0;
  		left: 0;
  		right: 0;
  		bottom: 0;
  	}
  \`);
`;

exports[`css module styles 1`] = `
  import css from "./somefile.module.css";
  import React from "react";
  () => <div className={css.element_7nq} />;
`;
